debian/jitsi-meet-turnserver.postinst

```diff
@@ -170,18 +150,14 @@ denied-peer-ip=240.0.0.0-255.255.255.255" >> $TURN_CONFIG
         invoke-rc.d coturn restart || true
 
         NGINX_STREAM_CONFIG="/etc/nginx/modules-enabled/60-jitsi-meet.conf"
-        if [ $NGINX_MULTIPLEXING = "true" ] && [ ! -f $NGINX_STREAM_CONFIG ] && [ -f $NGINX_CONFIG ] ; then
-            ln -s /usr/share/jitsi-meet-turnserver/jitsi-meet.conf $NGINX_STREAM_CONFIG
-            sed -i "s/listen 443 ssl/listen 4444 ssl http2/g" $NGINX_CONFIG
-            sed -i "s/listen \[\:\:\]\:443 ssl/listen \[\:\:\]\:4444 ssl http2/g" $NGINX_CONFIG
-            invoke-rc.d nginx reload || true
-        else
-            PROSODY_HOST_CONFIG="/etc/prosody/conf.avail/$JVB_HOSTNAME.cfg.lua"
-            if [ -f $PROSODY_HOST_CONFIG ] ; then
-                # If we are not multiplexing we need to change the port in prosody config
-                sed -i 's/"443"/"5349"/g' $PROSODY_HOST_CONFIG
-                invoke-rc.d prosody restart || true
-            fi
+        if [ -f $NGINX_STREAM_CONFIG ] ; then
+            echo "------------------------------------------------"
+            echo ""
+            echo "You have multiplexing enabled, it is recommended to disable it and migrate to using websockets for the bridge channel."
+            echo "The support for sctp data channels is deprecated and will be dropped at some point."
+            echo "How to do it at: https://jitsi.org/multiplexing-to-bridge-ws-howto"
```

Comment:
This 404s for me

Reply:
Cause I need to add it :) It is a work in progress

Reply:
And that is a redirect we can edit later if something comes up.

Reply:
Tha page is now live.